lab3/src/pages/home/index.css

```diff
@@ -0,0 +1,8 @@
+.calendar {
+  width: 380px;
+  margin-bottom: -30px;
```

Comment:
Negative margins are not acceptable.

Reply:
ok i will fix it but can u just explain me about negative margibs as i know bootstrap and tailwind are using negative margins

Reply:
It won't effect ur performance, but why u need negative margin if u can do it without? Generally the need of negative margins shows that u made a mistake somewhere before, which is not allowing u to position ur elements in the flow. It just make ur code more complicated and difficult to read for another developers. Yes, sometimes u can't do without them, if u are using some libraries and there is no way to customise their basic elementsl, but most of the time it's just a rubbish in ur code. The same as for JQuery, 10 years ago it was a very useful tool with some unique feature, for now u don't need additional 3gb for ur code as everything is in js rn.

Reply:
It won't effect ur performance, but why u need negative margin if u can do it without? Generally the need of negative margins shows that u made a mistake somewhere before, which is not allowing u to position ur elements in the flow. It just make ur code more complicated and difficult to read for another developers. Yes, sometimes u can't do without them, if u are using some libraries and there is no way to customise their basic elementsl, but most of the time it's just a rubbish in ur code. The same as for JQuery, 10 years ago it was a very useful tool with some unique feature, for now u don't need additional 3gb for ur code as everything is in js rn.

Reply:
ok

Reply:
fixed

Reply:
fixed